lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -540,6 +540,9 @@ export class UserAgentApplication {
 
       // prompt user for interaction
       this.navigateWindow(urlNavigate, popUpWindow);
+
+      const hash = await this.pollWindowForHash(popUpWindow);
```

Comment:
would this ever throw / catch?

Reply:
It will throw a timeout error if it doesn't find a hash after 6 seconds.

Reply:
Which will get caught in the `catch` block below.